.github/workflows/publish.yml

```diff
@@ -17,6 +20,7 @@ concurrency:
 jobs:
   build:
     runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true || github.event_name == "push" || github.event_name == "workflow_dispatch"
```

Comment:
I'm not sure about this part:
```
github.event_name == "workflow_dispatch"
```

`"workflow_dispatch"` might not be the right magic string.

Reply:
GitHub search seems to indicate it is correct, though: 🎉 
https://github.com/process-analytics/github-actions-playground/blob/master/.github/workflows/workflow-dispatch.yml#L22

Reply:
For future reference: it's a quote nonsense thing. If your variable contains quotes which need to survive, you need to wrap them in the other quote.

```yaml
thing: a quote # ok
thing: a "quote" # wrong
thing: 'a "quote"' # correct
thing: "a 'quote'" # also correct (if syntax requires single quote)
thing: >-
  'a "quote"' # preserves the outer quotes should you need both present
```

last I checked, GHA required the single quotes, so you _had_ to write it like `thing: "a 'quote'"` but I really hope that's not the case anymore.

Reply:
Thanks, @hazelweakly! We ended up getting that sorted out in #160 🎉 

Quote nonsense, indeed! It seems like `"string"` is just not valid in the expression evaluation:
https://docs.github.com/en/actions/learn-github-actions/expressions#literals

> Wrapping with double quotes (") will throw an error.

TIL! 😊